requirements/test.txt

```diff
@@ -14,6 +14,28 @@ requests==2.11.1 \
 py==1.4.31 \
     --hash=sha256:4a3e4f3000c123835ac39cab5ccc510642153bc47bc1f13e2bbb53039540ae69 \
     --hash=sha256:a6501963c725fc2554dabfece8ae9a8fb5e149c0ac0a42fd2b02c5c1c57fc114
+cssselect==0.9.1 \
+    --hash=sha256:0535a7e27014874b27ae3a4d33e8749e345bdfa62766195208b7996bf1100682
+lxml==3.5.0 \
+    --hash=sha256:f4253563021feb84aa2d4c06443418099b07356c59a3f4a77fd7eaf36d54642e \
+    --hash=sha256:2d46192109bf662413febb54fe3a3354d564bc274dd5454755b39d6a8f1f0697 \
+    --hash=sha256:f1ee626db1365fc789dc5b3807cb7ea50a0d004d60404288a97e9424daa45cfd \
+    --hash=sha256:3caadf90ba3f0442f4f75e8b6c5601b70bf17f94d41ded92232757de257ed084 \
+    --hash=sha256:3178c8fbe9a7b5356edf7d351fc97cd653527e2f90fe717ba42bdce24746137d \
+    --hash=sha256:349f93e3a4b09cc59418854ab8013d027d246757c51744bf20069bc89016f578 \
+    --hash=sha256:8628cc82957c41be10abce889a1976ceb7b9e3f36ebffa4fcb1a80901bf77adc \
+    --hash=sha256:1c9c26bb6c31c3d5b3c104e843211d9c105db60b4df6770ac42673263d55d494 \
+    --hash=sha256:793e7cfe96042fa0a11a99a70dbe61ab9275f268ea813606f63dccb288f3a1ee \
+    --hash=sha256:bb3264f80d31eb991cfe427cc20c83508eb326454c9a01a006829fa2f541be72 \
+    --hash=sha256:f396094325c1ade0c1aa69100c7e760faefafb362f17e858a531f903aff29937 \
+    --hash=sha256:01e54511034333f18772c335ec0b33a76bba988135eaf727a075897866d19604 \
+    --hash=sha256:2abf6cac9b7952047d8b7265384a9565e419a727dba675e83e4b7f5b7892b6bb \
+    --hash=sha256:6dff909020d0c030fb26004626c8f87f9116e0381702fed415caf94f5a9b9493 \
+    --hash=sha256:192fec5a960abf261bd3733a33c10c359ebd2f9a5b714a8191f382e8fa127d42 \
+    --hash=sha256:78a1ec1e674ab6dc8a0644bb4401f3fb5787b636b0aaf0c9a720923491547b1b
+pyquery==1.2.11 \
```

Comment:
If there is time, I'd like this to user [constraints.txt](https://github.com/mozilla/kuma/blob/master/requirements/constraints.txt) like the [default.txt](https://github.com/mozilla/kuma/blob/master/requirements/default.txt) requirements file.